doc/zh-cn/README.md

```
@@ -813,6 +814,103 @@ architecture-beta
 
 ![Architecture Diagram](../architecture/image.png)
 
+### 状态图语法
+
+```go
+package main
+
+import (
+	"io"
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)